Resources/Prototypes/_NF/Events/nf_events_bluespace_grids.yml

```diff
@@ -72,14 +102,24 @@
       path: /Audio/Misc/notice1.ogg
     endAnnouncement: station-event-bluespace-generic-ftl-end-announcement
     earliestStart: 80
-    minimumPlayers: 20
+    minimumPlayers: 15
     weight: 1
     duration: 1800
     maxDuration: 2400
     reoccurrenceDelay: 480 # Only once per shift possible
   - type: BluespaceErrorRule
-    gridPaths:
-      - /Maps/_NF/Bluespace/syndieftlintercept.yml
+    groups:
+      grid: !type:GridSpawnGroup
+        minimumDistance: 1500
+        maximumDistance: 2500
+        addComponents:
+        - type: Gravity
+          enabled: true
+          inherent: true
+        - type: IFF
+          color: "#E10F9B"
```

Comment:
Should also add comp to make the grid static (not in air) or heavy
Should also make sure it cannot have its mass fucked with by shuttle computers

Reply:
The latter part is fine, they aren't stations, so this shouldn't be an issue.